terraform/040-id-broker/vars.tf

```
@@ -238,26 +225,26 @@ variable "invite_lifespan" {
 }
 
 variable "lost_security_key_email_days" {
-  type    = string
-  default = "62"
+  type    = number
+  default = 62
 }
 
 variable "memory" {
-  type        = string
+  type        = number
   description = "Amount of memory to allocate to container, recommend '200' for production"
-  default     = "200"
+  default     = 200
 }
 
 variable "memory_cron" {
-  type        = string
+  type        = number
   description = "Amount of memory to allocate to cron container, recommend '200' for more than 500 active users"
-  default     = "200"
+  default     = 200
 }
 
 variable "memory_email" {
-  type        = string
+  type        = number
   description = "Amount of memory to allocate to email container"
```

Do we want recommend here?
```suggestion
  description = "Amount of memory to allocate to email container, recommend '64' for production"
```